### DISCUSSION TITLE
Will we be able to use WinUI 3 to develop for Xbox/Hololens/Windows Team?

### DISCUSSION THREAD

**Discussion by themerror** [microsoft/WindowsAppSDK#2759] [Q&A] [answered]
Hello!

As Microsoft had said before

> The Microsoft development story continues to evolve, and along with initiatives such as WinUI, MSIX, and the Windows App SDK, UWP is a powerful tool for creating client apps.

UWP would be a part of WASDK.

But now UWP support has been removed from the roadmap, and UWP will not have any update. So we have to develop on .Net Core 2.x.

If we want to use higher dotnet version to get better performance, we have to use WASDK, which cannot be used on Xbox.

So, will UWP be a part of WASDK again, and receive an update of dotnet? Or will UWP be abandoned and make WASDK be able to use on Xbox or other platforms?

Thanks!

**Comment by ghost** [comment, marked as answer]
https://github.com/microsoft/WindowsAppSDK/issues/215
https://github.com/microsoft/WindowsAppSDK/discussions/383

that was 2020, now is 2022. 

**Tl.Dr  :**    
For microsoft, Xbox/Hololens/Teams  marketshare isn't worth of consideration for supporting WASDK / WinUI 3 
   there.

**Reply by MartinZikmund** [reply]
It is very unfortunate seeing Xbox abandoned for app development. UWP was fantastic, but it is not possible to keep supporting UWP-based apps unless it gets proper support for .NET 7 or newer. There are great Xbox-optimized UWP apps like Ambie, which are used a lot on Xbox. If WinAppSDK gained support for Xbox, it would make it feature complete and a true UWP successor.
